tests/yang/YANGBearTest.py

```diff
@@ -0,0 +1,44 @@
+from bears.yang.YANGBear import YANGBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+VALID = """
```

Comment:
Please rename this.
same principle as https://github.com/coala/coala-bears/pull/1428#discussion_r110520711